data_tools/schema.py

```diff
@@ -0,0 +1,97 @@
+import validators
+
+class Schema(object):
+    scheme_id = None
+    name = None
+    version = None
+    codes = []
+    documentation = {}
+
+    def __eq__(self, other):
+        if not isinstance(other, self.__class__):
+            return False
+        return other.scheme_id == self.scheme_id & \
+            other.name == self.name & \
+            other.version == self.version & \
+            other.documentation == self.documentation & \
+            other.codes == self.codes
+    
+    def __neq__(self, other):
+        return not self.__eq__(other)
+
+
+    @staticmethod
+    def from_map(data):
```

Comment:
Wonder if `from_dict` (or even `load_dict`) is more Pythonic

Reply:
Done per discussion